Tests/Services/DeviceDetectTest.php

```diff
@@ -47,6 +51,35 @@ public function toggleWithAndWithoutAppConfig(): array
         ];
     }
 
+    /**
+     * Data provider to test multiple user agents:
+     * [user-agent, isDesktop, isMobile, isTablet]
+     *
+     * @return array
+     */
+    public function userAgentProvider(): array
+    {
+        return [
+            ['Mozilla/4.0 (compatible; MSIE 6.0; Windows NT 5.1)', true, false, false],
+            ['Mozilla/5.0 (Linux; Android 7.0; SM-G930V Build/NRD90M)', false, true, false],
+            ['Mozilla/5.0 (Linux; Android 4.4.3; KFTHWI Build/KTU84M)', false, false, true]
+        ];
+    }
+
+    /**
+     * Returns a RequestStack instance with the provided userAgent in the header.
+     *
+     * @param string $userAgent
+     * @return RequestStack
```

Comment:
```suggestion
     *
     * @return RequestStack
```
> Group annotations together so that annotations of the same type immediately follow each other, and annotations of a different type are separated by a single blank line;

Our current standard: https://symfony.com/doc/3.4/contributing/code/standards.html
Also present in the 4.0 version: https://symfony.com/doc/4.0/contributing/code/standards.html

Reply:
Ok, changed.